timelapse_tools/utils/czi_metadata.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python
```

Comment:
Will this file be moved to aicsimageio?

Reply:
I would hope a lot of this work will at some point make it's way in a more general form to both `aicsimageio` and `aicsimageprocessing`.

Reply:
I would hope that a lot of this work is moved to `aicsimageio` and `aicsimageprocessing` at some point soon.